src/lib/rnp.cpp

```diff
@@ -963,6 +963,29 @@ rnp_supported_features(const char *type, char **result)
     return ret;
 }
 
+rnp_result_t
+rnp_request_password(rnp_ffi_t ffi, rnp_key_handle_t key, const char *context, char **password)
+{
+    if (!ffi || !password || !ffi->getpasscb) {
+        return RNP_ERROR_NULL_POINTER;
+    }
+
+    Botan::secure_vector<char> pass(MAX_PASSWORD_LENGTH, '\0');
+    bool                       req_res =
+      ffi->getpasscb(ffi, ffi->getpasscb_ctx, key, context, pass.data(), pass.size());
+    size_t pass_len = strlen(pass.data());
+    if (!req_res || !strlen(pass.data())) {
```

Comment:
Can we just use `pass_len` here instead of calling `strlen` again?

Reply:
@dewyatt Sure, somehow missed this. Please see the updated PR.